yarn.lock

```diff
@@ -6185,12 +6185,12 @@ acorn@^6.4.1:
   resolved "https://registry.yarnpkg.com/acorn/-/acorn-6.4.1.tgz#531e58ba3f51b9dacb9a6646ca4debf5b14ca474"
   integrity sha512-ZVA9k326Nwrj3Cj9jlh3wGFutC2ZornPNARZwsNYqQYgN0EsV2d53w5RN/co65Ohn4sUAUtb1rSUAOD6XN9idA==
 
-acorn@^7.0.0:
+acorn@^7.0.0, acorn@^7.1.1:
```

Comment:
Why the yarn.lock change?

Reply:
Ran `yarn install` locally which updated the lock file. Most likely the lock file was outdated on `main`, which should've been caught on CI

Reply:
Gonna do a rebase to verify

Reply:
Yuup... this is supposed to be valid.